week1&week2/velog/src/write.js

```diff
@@ -0,0 +1,66 @@
+const ENTER = 13;
+
+const tagBox = document.querySelector(".tag-container");
+let tagSet = new Set();
+
+function saveLocalStorageData(item) {
+  localStorage.setItem("tagSet", JSON.stringify([...item]));
+}
+
+function checkSet(element, value) {
+  if (tagSet.has(value)) {
+    alert("다른거 써라");
+    return;
+  }
+  tagSet.add(value);
+  createTag(value);
+  createNextInput();
+  deletInput(element);
+  saveLocalStorageData(tagSet);
+}
+
+function handleEnter(element) {
+  element.addEventListener("keyup", (e) => {
+    if (e.keyCode === ENTER) {
+      checkSet(element, e.target.value);
+    }
+  });
+}
+
+function deleteTag(element) {
+  element.addEventListener("click", () => element.remove());
+}
+
+function deletInput(element) {
+  element.remove();
+}
+
+function createNextInput() {
+  const newInput = document.createElement("input");
+  tagBox.appendChild(newInput);
+  handleEnter(newInput);
+}
+
+function createTag(text) {
+  const tag = document.createElement("div");
+  tagBox.appendChild(tag);
+  tag.classList.add("tag");
+  tag.innerHTML = text;
+
+  deleteTag(tag);
+}
+
+function init() {
+  if (localStorage.length !== 0) {
```

Comment:
로컬스토리지에 뭐든 있어도 되는건가요 ??

아래는 어떨까 합니당

```suggestion
  if (localStorage.getItem("tagSet")) {
```

Reply:
홀리몰리 그렇네요 위험한 코드였네🚀 감사합니당!!!